.github/workflows/ci.yml

```diff
@@ -65,10 +103,10 @@ jobs:
         working-directory: ./sdk
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python ${{ env.X_PYTHON_VERSION }}
+    - name: Set up Python ${{ env.X_PYTHON_MIN_VERSION }}
```

Comment:
I guess, it also makes sense to do the static analysis with all supported Python versions, as in newer Python versions some varibles can be deprecated.
We can also do a matrix here.
I think it would be good to define a list of supported Python versions as `env` variable and use it for matrices.